solidity/app/contracts/AbacusConnectionClient.sol

```diff
@@ -13,7 +13,13 @@ abstract contract AbacusConnectionClient is OwnableUpgradeable {
     // ============ Mutable Storage ============
 
     IAbacusConnectionManager public abacusConnectionManager;
-    uint256[49] private __GAP; // gap for upgrade safety
+    // Interchain Gas Paymaster contract. The off-chain processor associated with
```

Comment:
should we s/processor/relayer?

Also wondering if we should make it clear in here that the interchain gas paymaster may present a reentrancy vector so the implementation should be confirmed by the application to not be able to re-enter. Also happy for this to just be a docs thing